src/pages/ResearcherProfile.js

```diff
@@ -592,52 +579,52 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
                       label({ id: 'lbl_profileLibraryCard', className: 'control-label' }, ['Library Cards']),
```

Comment:
LibraryCards aren't showing up on the profile page. This could be another outdated feature of the page (I remember it manually fetched LCs with an array of user ids.) Since the profile now gets library cards directly we can just use it directly.

<img width="1662" alt="Screen Shot 2021-08-19 at 8 56 25 AM" src="https://user-images.githubusercontent.com/6664778/130072522-51033a0c-6857-47e3-ab18-7edae323565f.png">